### DISCUSSION TITLE
Layer selection & view options

### DISCUSSION THREAD

**Discussion by Tempy111** [arkypita/LaserGRBL#2103] [Ideas]
When you add a file, you have no control over it once it's been converted. atleast giving the option to see the feed rate/power settings might be useful

also when you append a second file, the script will always run in the order it's been added. having some control so you can 'relayer' bits might be useful.

**Comment by Tribo-2dot0** [comment]
No rodapé, ao lado da pausa, você tem alguns controles que podem ser usados durante o processo.
![Captura de tela 2023-07-20 122502](https://github.com/arkypita/LaserGRBL/assets/83982525/e581d293-2c7a-4297-9166-585b4b1731ba)
